### DISCUSSION TITLE
Complex query examples

### DISCUSSION THREAD

**Discussion by anch-anch** [comunica/comunica#1132] [Q&A] [answered]
I would like to understand how we can write more complex analytical queries in GQL in a similar fashion that SPARQL allows us to query for relationships between nodes and properties or for entities that have specific properties. I read the documentation and there are some examples there however are there verbose schema examples somewhere that shows RDF/TTL stores being queried via GraphQL in depth?

**Comment by rubensworks** [comment, marked as answer]
GraphQL(-LD) is not as expressive as SPARQL. So if you need to express complex questions, then I recommend using SPARQL.

(FYI, [GQL](https://www.gqlstandards.org/) != [GraphQL](https://graphql.org/))
